Implement prototype audio recorder.

max-vogler opened the pull request:
See #205.

hoch commented:
Looks great! Can we have minimum comments on the JS file? Are you using a linter?

hoch reviewed (commented):
I also suggest adding a very simple README.md to explain what it does and how it is done.

max-vogler commented:
> Looks great! Can we have minimum comments on the JS file? Are you using a linter?

Added comments. All code is formatted with prettier.io.

> I also suggest adding a very simple README.md to explain what it does and how it is done.

Added README.

max-vogler commented on audio-recorder/app.js:
```diff
@@ -0,0 +1,112 @@
+"use strict";
```
I use prettier.io which defaults to double-quotes. Shall I add a configuration file to override it, or leave it at double quotes?

max-vogler commented on audio-recorder/app.js:
```diff
@@ -0,0 +1,112 @@
+
+    recordOutlineEl.style.boxShadow = `0 0 0 ${
+      volume * 20
+    }px rgba(0, 0, 0, 0.2)`;
```
Yes. I extracted `volume * 20` into a variable and now Prettier leaves the format string in one line.

max-vogler commented on audio-recorder/app.js:
```diff
@@ -0,0 +1,112 @@
+
+  source.connect(analyser);
+
+  let x = -1;
```
Added.

hoch commented:
Perhaps I should've shared this with you earlier. FYI:
https://github.com/GoogleChrome/samples#style--linting--ci

For future contribution, let us follow the guideline. This PR looks great as is.
